net/net-ifnet.cpp

```diff
@@ -75,7 +75,7 @@ unsigned get_my_ipv4() {
     }
     unsigned ip = ntohl(((struct sockaddr_in *)ifa->ifa_addr)->sin_addr.s_addr);
     unsigned mask = ntohl(((struct sockaddr_in *)ifa->ifa_netmask)->sin_addr.s_addr);
-    vkprintf (2, "%08x %08x (%08x == %08x)\t%s\t-> %d,%d\n", ip, mask, (ip & force_ipv4_mask), force_ipv4_ip, ifa->ifa_name, (ip & force_ipv4_mask) == force_ipv4_ip, mask < my_netmask);
+    vkprintf (4, "%08x %08x (%08x == %08x)\t%s\t-> %d,%d\n", ip, mask, (ip & force_ipv4_mask), force_ipv4_ip, ifa->ifa_name, (ip & force_ipv4_mask) == force_ipv4_ip, mask < my_netmask);
```

Comment:
This message is printed only once during start, I think it's better to lower verbosity guard for it